packages/cli/src/commands/org/config/pull.ts

```diff
@@ -17,20 +18,26 @@ export default class Pull extends Command {
   public static flags = {
     target: flags.string({
       char: 't',
-      description:
-        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
       helpValue: 'destinationorganizationg7dg3gd',
       required: false,
+      description:
+        'The unique identifier of the organization from which to pull the resources. If not specified, the organization you are connected to will be used.',
     }),
-    // TODO: CDX-447: add flag to select resource types to export
+    resourceTypes: flags.string({
```

Comment:
Is `ResourceType` a string enum in platform client ? 

If so, I think using `flags.enum` would be more appropriate, and would remove the need to cast on line 33. 

You also wouldn't need to do any mapping on line 62-63, since you would already have the correct value I think.

Reply:
I tried with enum but looks like `flags.enum` only accept one value

Reply:
Like this here ? https://github.com/coveo/cli/blob/master/packages/cli/src/commands/source/push/new.ts#L19

Reply:
Ohh I understand, you can't repeat.

NVM me then.